app.rb

```diff
@@ -3,7 +3,17 @@
 module FormsLab
   class App < Sinatra::Base
 
-    # code other routes/actions here
+    get "/" do
+      erb :root
+    end
 
+    get "/new" do
+      erb :'pirates/new'
+    end
+
+    post "/pirates" do
+      @pirate = params
+      erb :'pirates/show'
+    end
```

Comment:
Your POST action should also contain logic to create a new instance of a pirate and its associated ships. This would make it easier to handle for the show page. We'll go over this in more detail on Monday.